lib/resources/file.rb

```diff
@@ -133,6 +133,35 @@ def sticky
 
     alias sticky? sticky
 
+    def more_permissive_than?(max_mode = nil)
+      raise Inspec::Exceptions::ResourceFailed, 'The file' + file.path + 'doesn\'t seem to exist' unless exist?
+      raise ArgumentError, 'You must provide a value for the `maximum permission target` for the file.' if max_mode.nil?
```

Comment:
Consider `maximum allowable permission`. `target` has a special meaning to InSpec users.